conductor/resources/maya.ui

```diff
@@ -24,23 +33,40 @@
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
-       <widget class="QCheckBox" name="ui_upload_only">
-        <property name="toolTip">
-         <string>Only upload dependencies. Don't submit render job</string>
-        </property>
-        <property name="text">
-         <string>Upload Only</string>
-        </property>
-       </widget>
+       <layout class="QHBoxLayout" name="horizontalLayout">
+        <item>
+         <widget class="QCheckBox" name="ui_upload_only">
+          <property name="toolTip">
+           <string>Only upload dependencies. Don't submit render job</string>
+          </property>
+          <property name="text">
+           <string>Upload Only</string>
+          </property>
+         </widget>
+        </item>
+        <item>
+         <widget class="QLabel" name="ui_render_log_label">
+          <property name="maximumSize">
+           <size>
+            <width>50</width>
+            <height>16777215</height>
+           </size>
+          </property>
+          <property name="text">
+           <string>Log Level:</string>
+          </property>
+         </widget>
+        </item>
+        <item>
+         <widget class="QComboBox" name="ui_render_log_level_cbox"/>
```

Comment:
Yes, it could go to Advanced. 
It does fit quite tidily there under render layers too. 
We also have a different log level choice in the Options menu. Do they both have annotation popups? Maybe they could be grouped.

You could have a string attribute somewhere and allow users to add arbitrary extra flags (and no doubt screw up their renders 9 times out of 10). I notice Arnold does this with aiUserOptions. Definitely advanced.

Reply:
@lawschlosser You're right, this change should have required a refactor. Since I know an API and a redo of the submitter is coming, I thought it wasn't worth the effort.

Reply:
@lawschlosser I've pushed an update where I address this